x/staking/teststaking/tm.go

```diff
@@ -19,21 +20,21 @@ func GetTmConsPubKey(v types.Validator) (tmcrypto.PubKey, error) {
 }
 
 // ToTmValidator casts an SDK validator to a tendermint type Validator.
-func ToTmValidator(v types.Validator) (*tmtypes.Validator, error) {
+func ToTmValidator(v types.Validator, powerReduction sdk.Int) (*tmtypes.Validator, error) {
```

Comment:
validator param name is `v` shortest as possible, recommend using `r` for reduction.